specification/src/main/asciidoc/platform/Interoperability.adoc

```diff
@@ -77,27 +77,27 @@ transport protocols for the Internet. TCP/IP is supported by Java SE and
 the underlying operating system.
 * HTTP 1.1—This is the core protocol of web
 communication. As with TCP/IP, HTTP 1.1 is supported by Java SE and the
-underlying operating system. A Java EE web container must be capable of
+underlying operating system. A Jakarta EE web container must be capable of
 advertising its HTTP services on the standard HTTP port, port 80.
 * HTTP/2—Server-side support for the HTTP/2
 protocol is required by the Servlet specification.
 * SSL 3.0, TLS 1.2—SSL 3.0 (Secure Socket Layer)
 represents the security layer for Web communication. It is available
 indirectly when using the _https_ URL as opposed to the _http_ URL. A
-Java EE web container must be capable of advertising its HTTPS service
+Jakarta EE web container must be capable of advertising its HTTPS service
 on the standard HTTPS port, port 443. Note that SSL 3.0 and only TLS 1.0
-are required as part of the EJB interoperability protocol in the EJB
+are required as part of the Jakarta Enterprise Beans interoperability protocol in the Jakarta Enterprise Beans
 specification.
 * SOAP 1.1—SOAP is a presentation layer
 protocol for the exchange of XML messages. Support for SOAP layered on
-HTTP is required, as described in the JAX-RPC and JAX-WS specifications.
+HTTP is required, as described in the Jakarta XML-based RPC and Jakarta XML Web Services specifications.
```

Comment:
Jakarta XML Web Services is not part of Jakarta EE 8.
This should continue to refer to JAX-WS.